pom.xml

```diff
@@ -6,7 +6,7 @@
     <parent>
         <groupId>org.mule.extensions</groupId>
         <artifactId>mule-core-modules-parent</artifactId>
-        <version>1.1.0</version>
+        <version>1.1.2-SNAPSHOT</version>
```

Comment:
This should only be changed for testing purpouses.

Reply:
Keep the `1.1.0` version, we can't do this because it will change the Min Mule Version that the connector supports.
It's sucks because the flaky tests will still be failing, but there are no other alternatives than migrate those tests to MTF.